app/build.gradle

```diff
@@ -63,7 +63,11 @@ dependencies {
     // Hilt
     implementation "com.google.dagger:hilt-android:2.38.1"
     kapt "com.google.dagger:hilt-compiler:2.38.1"
+    kapt "androidx.hilt:hilt-compiler:1.0.0"
```

Comment:
Question: Why there are 2 kapt hilt compilers? 
https://stackoverflow.com/questions/62882730/the-hilt-android-gradle-plugin-is-applied-but-no-com-google-daggerhilt-android

Reply:
> Question: Why there are 2 kapt hilt compilers?
> https://stackoverflow.com/questions/62882730/the-hilt-android-gradle-plugin-is-applied-but-no-com-google-daggerhilt-android

![Screen Shot 2021-11-18 at 12 55 56 PM](https://user-images.githubusercontent.com/43578259/142393496-1fd3d9a6-be50-4342-9e79-3c0003d4db5b.png)

And this is not helping to unite two  kapt hilt compilers into one
![Screen Shot 2021-11-18 at 12 59 32 PM](https://user-images.githubusercontent.com/43578259/142393646-038d0e09-2251-4f9d-b846-4b6b01906e64.png)